spec_helper.rb

```diff
@@ -14,9 +14,20 @@ module SpecHelper
 unless ENV["DEBUG"]
   fastlane_tests_tmpdir = "#{Dir.tmpdir}/fastlane_tests"
   $stdout.puts("Changing stdout to #{fastlane_tests_tmpdir}, set `DEBUG` environment variable to print to stdout (e.g. when using `pry`)")
+  $orig_stdout = $stdout
   $stdout = File.open(fastlane_tests_tmpdir, "w")
 end
 
+# for troubleshooting CI issues on specific tests
+def with_orig_stdout(&block)
```

Comment:
Maybe add docs on how to use this? It's not clear to me when or how I'd use this

Reply:
documented. Makes sense?